felix/bpf-gpl/tc6.c

```diff
@@ -89,13 +87,19 @@ int calico_tc(struct __sk_buff *skb)
 	CALI_DEBUG("IPv6 on host interface: allow\n");
 	CALI_DEBUG("About to jump to normal policy program\n");
 	CALI_JUMP_TO(skb, PROG_INDEX_V6_POLICY);
```

Comment:
QQ: do we have access to parameters that could be added to the DEBUG statements i.e. the host interface that could maybe help with future debugging?

Reply:
CALI_DEBUG reports the interface. BPF supports only 5 args to a helper function. 2 are taken by the format and its length and 1 by the iface,so we only have 2 remaining  so we have to be careful with how much context we add automatically :(